BossMod/BossModule/AIHints.cs

```diff
@@ -90,16 +90,24 @@ public void FillPotentialTargets(WorldState ws, bool playerIsDefaultTank)
     {
         bool playerInFate = ws.Client.ActiveFate.ID != 0 && ws.Party.Player()?.Level <= Service.LuminaRow<Lumina.Excel.GeneratedSheets.Fate>(ws.Client.ActiveFate.ID)?.ClassJobLevelMax;
         var allowedFateID = playerInFate ? ws.Client.ActiveFate.ID : 0;
-        foreach (var actor in ws.Actors.Where(a => a.Type == ActorType.Enemy && a.IsTargetable && !a.IsAlly && !a.IsDead))
+        foreach (var actor in ws.Actors.Where(a => a.Type is ActorType.Enemy or ActorType.Part && a.IsTargetable && !a.IsAlly && !a.IsDead))
         {
             // fate mob in fate we are NOT a part of, skip entirely. it's okay to "attack" these (i.e., they won't be added as forbidden targets) because we can't even hit them
             // (though aggro'd mobs will continue attacking us after we unsync, but who really cares)
             if (actor.FateID > 0 && actor.FateID != allowedFateID)
                 continue;
+            
+            // target is dying; skip it so that AI retargets, but ensure that it's not marked as a forbidden target
+            // skip this check on striking dummies as they die constantly
+            var predictedHP = ws.PendingEffects.PendingHPDifference(actor.InstanceID);
+            if (actor.HPMP.CurHP + predictedHP <= 0 && actor.OID != 0x23B)
```

Comment:
Aren't there a whole bunch of different striking dummies with different id's?

Reply:
yeah good point, we could check for nameID == 541

Reply:
Well do they all have the same nameid? :D

Reply:
Sheets only has "striking dummy" (541) and "timeworn striking dummy" (13078) (no idea what this is even)

the dummy at my house, the one in explorer mode, and the ones at Falcon's Nest in coerthas are all 541, so i'm gonna assume it applies to all